src/osd/PG.h

```diff
@@ -170,6 +171,7 @@ class PG : public DoutPrefixProvider,
   friend class PeeringState;
   friend class PgScrubber;
   friend class ScrubBackend;
+  friend class ScrubQueue;
```

Comment:
Why?  It seems like PG should be a normal consumer of ScrubQueue's public interface.